ambry-api/src/main/java/com.github.ambry/config/ClusterMapConfig.java

```diff
@@ -231,6 +232,20 @@
   @Default("false")
   public final boolean clustermapUpdateDatanodeInfo;
 
+  /**
+   * Partition layout json for recovery test cluster map.
+   */
+  @Config("clustermap.recovery.test.partition.layout")
+  @Default("{}")
+  public final JSONObject clustermapRecoveryTestPartitionLayout;
```

Comment:
These things look out of place here. 
Also, if the cluster agents factories are constructed with the two layout files, why do we even need the config properties?